tests/acceptance/features/webUIResharing1/reshareUsers.feature

```diff
@@ -162,7 +162,7 @@ Feature: Resharing shared files with different permissions
       | item_type   | folder                |
       | permissions | share, delete, read   |
 
-  @issue-5461
+  @skipOnOCIS
```

Comment:
This test is skipped because as mentioned in the issue https://github.com/owncloud/web/issues/6896, it also has a similar workflow of opening the Shares folder through UI and doing some action. Previously this test was expected to fail but they were linked to a closed issue.